packages/flutter/lib/src/painting/debug.dart

```diff
@@ -31,3 +31,13 @@ bool debugAssertAllPaintingVarsUnset(String reason, { bool debugDisableShadowsOv
   }());
   return true;
 }
+
+/// Whether [NetworkImage] should use a newly instantiated [HttpClient] when loading
+/// images.
+///
+/// This setting can be overridden for testing to ensure that each test receives a
+/// mock client that hasn't been affected by other tests.
+///
+/// This value is ignored in non-debug builds.
+@visibleForTesting
```

Comment:
why is this one `@visibleForTesting` but not the others?

Seems like it would be reasonable for this one to be usable from non-test debug code too?

Reply:
I wondered about that - I couldn't think of a concrete use case during debugging.  But happy to remove the annotation - will do.